### ISSUE TITLE
Fetch market price using web3

### ISSUE THREAD

**Issue by pi0neerpat** [open-dollar/od-sdk#54]
Depends on #53 

- Swap out the use of `oracleRelayer.marketPrice()` to get the ODG price, for the actual camelotRelayer we will deploy (depends on https://github.com/open-dollar/od-relayer/issues/33)
- Get the correct price for WSTETH using the chainlink oracle

**Comment by jahabeebs** [comment]
Closing as won't do -- part of TVL project
